src/registry/coverage.ts

```diff
@@ -27,10 +27,15 @@ type ApiVersion = {
   version: string;
 };
 
+let apiVer: number;
+
 export const getCurrentApiVersion = async (): Promise<number> => {
-  const apiVersionsUrl = 'https://dx-extended-coverage.my.salesforce-sites.com/services/data';
-  const lastVersionEntry = (await got(getProxiedOptions(apiVersionsUrl)).json<ApiVersion[]>()).pop() as ApiVersion;
-  return +lastVersionEntry.version;
+  if (apiVer === undefined) {
+    const apiVersionsUrl = 'https://appexchange.salesforce.com/services/data';
+    const lastVersionEntry = (await got(getProxiedOptions(apiVersionsUrl)).json<ApiVersion[]>()).pop() as ApiVersion;
```

Comment:
have a mild preference for `.at(-1)` over `.pop()` to avoid using the mutating Array methods (fine here, no side effects, just more of "if you never use it, you don't have to think about whether this time is safe").